cli/azd/pkg/auth/manager.go

```diff
@@ -432,14 +440,24 @@ func (m *Manager) LoginWithDeviceCode(
 		return nil, err
 	}
 
-	// Display the message to the end user as to what to do next, then block waiting for them to complete
-	// the flow.
-	fmt.Fprintln(deviceCodeWriter, code.Message())
+	m.console.MessageUxItem(ctx, &ux.MultilineMessage{
+		Lines: []string{
+			fmt.Sprintf("Start by copying the next code: %s", output.WithBold(code.UserCode())),
+			"Then press enter and continue to log in from your browser...",
+		},
```

Comment:
Do we have this wording reviewed by Austin? I think it can be improved upon.

Reply:
I'm taking @savannahostrowski approval as confirmation for wording.
We can always come back and update it